Contributing-to-Mastodon/Translating.md

```diff
@@ -46,3 +46,25 @@ Once the pull request is accepted, wait for the code to be deployed on a Mastodo
 ## Updating the translation
 
 Keep an eye on the original English files in `app/assets/javascripts/components/locales` and `config/locales`. When they are updated, pass on the changes to your language files. For new strings, add the new lines to the same position and translate them. Once you’re finished with the updates, you can submit a new pull request.
+
+## Appendix A. Plural handling
+
+Different languages use different plural forms to be taken care of by Mastodon.
+
+For JavaScipt (`.jsx`) translations, this is done in [react-intl](https://github.com/yahoo/react-intl) way, by doing:
+
+```
+Here {appleCount, plural, one {is an apple} other {are {appleCount} apples}}.
+```
+
+On the other hand, `.yml` files are processed by [rails-i18n](https://github.com/svenfuchs/rails-i18n). Items that look like this are pluralized fields:
+
+```YML
+eat_apple:
+  one: You ate an apple.
+  other: You ate %{count} apples.
+```
+
+In both cases you can see a `one` case and an `other` case described for the pluralized strings. The exact strings is chosen by how many a certain quantity is -- when there is exactly one of something, the sentence goes to the `one` case; otherwise it goes to the `other` case. This is the case for English (`en`) and a few other languages.
```

Comment:
I am suffering from semantic saturation around the multiple instances (fine, cases) of "case" here. Any suggestions for a fix?